packages/components/package-lock.json

```diff
@@ -13,12 +13,6 @@
         "regenerator-runtime": "^0.13.4"
       }
     },
-    "@goodhood/icons": {
```

Comment:
why remove oodhood/icons?

Reply:
i assume because we moved it to root package some time ago